packages/design-core/README.md

```
@@ -0,0 +1,129 @@
+🌈 Features:
+
+- Cross-end cross-frame front-end components
+- Supports online real-time construction, secondary development, or being integrated.
+- Directly generate deployable source code without engine support.
+- Allows access to third-party components and customized extension plug-ins.
+- Supports high-code and low-code, and hybrid development and deployment of applications.
+- The platform accesses LLM capabilities to help developers build applications.
+
+## Documentation
+
+- intro：https://opentiny.design/tiny-engine#/home
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)